modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -94,6 +94,48 @@ def create
 
         private
 
+        def validate_decide_params!(proc_id:, decision:)
+          if proc_id.blank?
+            raise ::Common::Exceptions::ParameterMissing.new('procId',
+                                                             detail: 'procId is required')
+          end
+
+          unless decision.present? && %w[accepted declined].include?(decision)
+            raise ::Common::Exceptions::ParameterMissing.new(
+              'decision',
+              detail: 'decision is required and must be either "ACCEPTED" or "DECLINED"'
+            )
+          end
+        end
+
+        def send_declined_notification(ptcpnt_id:, first_name:, poa_code:)
+          lockbox = Lockbox.new(key: Settings.lockbox.master_key)
+          encrypted_ptcpnt_id = lockbox.encrypt(ptcpnt_id)
+          encrypted_first_name = lockbox.encrypt(first_name)
+
+          ClaimsApi::VANotifyDeclinedJob.perform_async(encrypted_ptcpnt_id:, encrypted_first_name:, poa_code:)
+        end
+
+        def validate_ptcpnt_id!(ptcpnt_id:, proc_id:, service:)
+          if ptcpnt_id.blank?
+            raise ::Common::Exceptions::ParameterMissing.new('ptcpntId',
+                                                             detail: 'ptcpntId is required if decision is declined')
+          end
+
+          res = service.read_poa_request_by_ptcpnt_id(ptcpnt_id:)
```

Comment:
meaning we could also make this call in the job so we don't have to send first name and poa code to the decline job? 

https://github.com/department-of-veterans-affairs/vets-api/pull/19362/files#r1842665735